Suggested charter edits and additions (#255)
todogroup/governance: GM-SC-CHARTER.adoc

= Charter of The TODO Group General Members (GM) and Steering Committee (SC)

> 

Reviewer: ```suggestion
*Disclaimer: Please note that this charter only covers the rights, criteria, and responsibilities of general members and the steering committee. It does not address other actors within the TODO Group, such as OSPO associate organizations and the individual community of contributors and OSPO practitioners. For information regarding these additional groups, please consult the [TODO Group community & network page](https://todogroup.org/community/)*
```

Reviewer 2: I think I know where you are going with this and why, but this disclaimer might be misleading rather than clarifying. For instance, it is not entirely true and accurate: The charter also talks about IP Policy (section 10), which applies to all participants, not just members. The same should be true for some of the operating principles, e.g., meetings are under Chatham House rules - for all I hope, not just members.

So, lacking a better suggestion right now, I'd rather opt not to include this.

Reviewer 3: Perhaps this is a better wording:

Please note this charter covers the activities and responsibilities of general members and the steering committee. There are many other folks active in TODO Group, such as individuals employed by OSPO Associate Organizations and individuals who are OSPO practitioners and contributors to open source projects and communities.  Individuals persons are welcome and encouraged to join and participate in the TODO Group as long as they abide by our code of conduct. More information on the [OSPO Associates](https://todogroup.org/associates/) page.

--- 
Side note, I am delighted we've arrived at this wording and made it abundantly clear that you do not have to be employed to participate in TODO. That was a big oversight and I am grateful to @annania for bringing it to our attention. Making this change is particularly timely given current market conditions and their impact to OSPO professionals.

 ... In each calendar year, the committee shall appoint one (1) member to fill the appointed position which will be vacated at the end of that year. The appointment shall be made during the month of October, by the current members of the commitee at that time.
 ... In even-numbered years, two (2) elected positions and one (1) appointed position shall be filled by the election and appointment processes.
 ... In odd-numbered years, three (3) elected positions and one (1) appointed position shall be filled by the election and appointment processes.
 ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members may appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
 ... If any committee member should resign or otherwise be unable to complete their term for any reason, with six (6) months or fewer remaining in their term, their position will be left unfilled until the regularly scheduled election or appointment process for that position.

>  ... Elections shall be run using the Condorcet-IRV method through the Cornell online service (+++<u>+++http://civs.cs.cornell.edu/+++</u>+++). The methodology or service used in future elections may be changed via a five (5) member approval vote of current members of the committee at that time.

Reviewer: ```suggestion
 ... Elections shall be run using the Condorcet-IRV method through the Cornell online service (+++<u>+++http://civs.cs.cornell.edu/+++</u>+++). The methodology or service used in future elections may be changed via a five (5) member approval vote of current members of the committee at that time. TODO Steering Committee members are elected for a two year term.
```

Reviewer 2: This addition should not be necessary because there is a bullet above stating "Committee member terms shall begin on January 1st of the year following election or appointment, and terminate on December 31st of the second year following election or appointment." If we want to state the 2-year term lengths explicitly, I'd add it before the sentence above.

Reviewer 3: I think it never hurts to say the same thing in different places. Agree with @gkunz on where to position this sentence. With that change, LGTM, please merge.

 . *Working Groups*
The TODO Group will host Working Groups for everyone, including its general members to contribute to that are intended to accelerate its mission. Initial Working Groups are expected to cover topics including, but not limited to:
 * Well documented software to facilitate the management of open source software engagement and operations by an organization
 * Blog and wiki articles providing recommendations on how to manage open source operations and implement open source usage and contributions best practices by an organization
 . *Operating Principles*
 .. TODO member meeting mechanisms and expected frequency
 ... General Member meetings may take place via electronic means or face-to-face. Face-to-face meeting locations and times will be set by the Steering Committee.
 ... General Member Meetings will occur at least once per quarter.
 ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all general members in good standing.
 .. Communication mechanisms

>  ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike.

Reviewer: ```suggestion
 ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike. Public communication must include all technical discussions and communications. Technical decisions of the project must not be done privately.
```

Reviewer: Technical is everything related to contributing/participating in TODO project repos + meetings related to the activity of each of the repos. For instance:

* [TODO Guides are done via todogroup.org](https://github.com/todogroup/todogroup.org) and the readme file clarifies how people can contribute and start a new guide or make changes to existing ones. 
* OSPOlogy has a [contribution ladder](https://github.com/todogroup/ospology/blob/main/GOVERNANCE.md#ospology-repo-governance) to know how people can become maintainers
*  People can read the [community page](https://todogroup.org/community/) to join any of the working groups that are active, communication channels, and mailing lists
* Announcements of project releases, like the OSPO mind map are shared via public [GH OSPOlogy Discussions](https://github.com/todogroup/ospology/discussions).

Maybe we can be more specific on the kind of topics that refers to technical in the playbook, and leave a short sentence like the one suggested here?

>  ... All private communication not during meetings will take place over the private group mailing list, real-time chats, issue trackers and source code management systems, and access will be limited to members only.

Reviewer: ```suggestion
 ... All private communication not during general member meetings will take place over (1) the general member mailing list and (2) real-time chat #private-generalmember channel and access will be limited to general members. Every issue tracker, repo documentation and source code management under TODO GitHub organization is considered technical communications and thus must be always public and part of public communications.
```

Reviewer 3: Agree with Georg's suggested changes.

 .. Level of confidentiality
 ... All communication during meetings will be under+++<u>+++ Chatham House Rules+++</u>++++++<u>+++.+++</u>+++
 ... All private communication via all media as defined above will be restricted to members only and not permitted for public distribution.
 ... All communication via all media as defined above will be public.
 . *Antitrust Guidelines*
 .. All members shall abide by The Linux Foundation Antitrust Policy available at: https://www.linuxfoundation.org/antitrust-policy/.

>  .. All members may encourage open participation from any organization able to meet the membership requirements, regardless of competitive interests. Put another way, the TODO Group shall not seek to exclude members based on any criteria, requirements, or reasons other than those used for all members.

Reviewer: ```suggestion
 .. All members shall encourage open participation from any organization able to meet the membership requirements, regardless of competitive interests. Put another way, the TODO Group shall not seek to exclude members based on any criteria, requirements, or reasons other than those used for all members.
```
